tests/test_server.py

```diff
@@ -138,6 +140,7 @@ def test_stop(repo):
         text=True,
         check=False,
     )
+    assert not psutil.pid_exists(server_info["pid"])
```

Comment:
I would make the opposite assertion (`assert psutil.pid_exists(server_info["pid"])`) before running the stop feature to make sure that it was running to begin with and that the method of checking it is correct

Reply:
Good idea! 👍